### ISSUE TITLE
Video stuck and no sound when using passthrough

### ISSUE THREAD

**Issue by aas88keyz** [CastagnaIT/plugin.video.netflix#296]
#### Your Environment
- Netflix add-on version: 15.8
- Operating system version/name: Android 9.0
- Device model:  Shield TV 2017

Used Operating system:
* [x] Android
* [ ] iOS
* [ ] Linux
* [ ] OSX
* [ ] Raspberry-Pi
* [ ] Windows

### Describe the bug

[kodi.log](https://github.com/CastagnaIT/plugin.video.netflix/files/3834078/kodi.log)

When playing a video through netflix plugin with the full ESN in place I can play 1080p and 4k  with no sound for a few seconds and the picture freezes. When going to the specific video sound settings and unclicking passthrough the video plays in the appropriate resolution with 2.0 dd+ sound. Video will also play when dd+ is turned off in the netflix plugin settings. Issue with passthrough. Also I don't know if it is notable in the kodi log but I can confirm that system setting "sync playback to display" is disabled and always is.

#### Expected behavior

Netlfix should play 4k and 1080p video and passthrough DD+ or DD Atmos on Nvidia Shield with the proper esn in place.

#### Actual behavior

Playing the netflix video plays no sound and video locks up but disabling passthrough  in the video settings plays the video in stereo.
#### Steps to reproduce the behavior

1. Start kodi
2. Find video in the kodi library menu
3. Click video to play
4. Video plays with no sound for 5 seconds
5. Video locks up
6. Access audio settings from within video and disable passthrough
7. Video plays in 1080p or 4k with 2.0 sound
8. Confirm system setting "sync playback to display" is disabled

### Debug log

The debug log can be found here: 

#### Installation
* [ ] I'm using other Netflix Repo 
* [ ] I'm using a different source

**Comment by CastagnaIT** [comment]
i don't have a device to test but seem there is wrong settings somewere which causes driver error

check Kodi system settings:
In Audio: turn OFF upmix, check passthrough is turn ON
In Video: sync to display to OFF, adjust display refresh rate to ON

try also remove advancedsettings

**Comment by aas88keyz** [comment]
Confirmed upmix OFF; passthrough ON; sync to display OFF; adjust display refresh rate ON start/stop
Removed advancedsettings.

Result is still no sound and video freezes after 5 seconds.
new log attached:
[kodi.log](https://github.com/CastagnaIT/plugin.video.netflix/files/3837800/kodi.log)

**Comment by robofunk** [comment]
You should try increasing the "Delay in Playback on refresh rate change" setting as well.

**Comment by aas88keyz** [comment]
ok I set delay for 3.0 seconds. Same status. should I delay more? Do you need another log?

**Comment by aas88keyz** [comment]
If it helps any I turned of HDR in netlfix settings but no improvement

**Comment by CastagnaIT** [comment]
already tried to reinstall kodi?
unfortunately they are not able to help you more, this is not a problem that concerns the netflix addon errors relate to Kodi system with the device, or your TV configuration or the cable is not good
I don't know what's wrong, maybe in the Kodi forum someone can help you better

**Comment by robofunk** [comment]
This may have been fixed in Kodi recently (https://github.com/xbmc/xbmc/pull/14604). You should try the Kodi Leia nighties (https://mirrors.kodi.tv/nightlies/android/arm64-v8a/Leia/).

**Comment by aas88keyz** [comment]
18.4 leia Nov 11 nightly seems to be the resolution. I am close anyway. Netflix will play the video in 4k HDR with no sound. I will continue my trouble shooting but will close the issue as I am very close. @robofunk thank you for your reference of https://github.com/xbmc/xbmc/pull/14604. The version I was using was from a couple days ago.  This put me in the right direction.

**Comment by robofunk** [comment]
> 18.4 leia Nov 11 nightly seems to be the resolution. I am close anyway. Netflix will play the video in 4k HDR with no sound. I will continue my trouble shooting but will close the issue as I am very close. @robofunk thank you for your reference of [xbmc/xbmc#14604](https://github.com/xbmc/xbmc/pull/14604). The version I was using was from a couple days ago. This put me in the right direction.

I read your log and thought you did where on a recent nightly that's why I deleted my message. Guess I was right after all.

**Comment by M2k80** [comment]
with shield 2017 and passtrough, you will get no sound if your AVR or soundbar does not support DD+ 
the shield 2017 has no dolby decoder

**Comment by aas88keyz** [comment]
Good to know. My AVR is a Yamaha TSR-5810. I recently confirmed it supports these bitstreams Dolby Digital, DTS, Dolby Digital Plus, DTS-HD  Master, DTS-HD High Resolution and DTS-HD Express. After updating to the 11/12/19 nightly Kodi Leia 18.4 version with a completely clean instal (no .kodi directory) and manually place the ESN in the Netflix addon, I was able to get video playing for 4k HDR videos but no sound. After unplugging all devices I booted one device at a time starting with TV to AVR to Shield. I started Kodi and played "Dark Crystal" with 4k HDR and Dolby atmos. Things are good. Thanks for everyone's help
